orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/pipeline/support/ResizeStrategy.groovy

```diff
@@ -52,6 +52,28 @@ interface ResizeStrategy {
     Integer min
   }
 
+  @Canonical
+  static class Source implements Serializable {
+    Collection<String> zones
+    Collection<String> regions
+    String region
+    String zone
```

Comment:
Seems like a single (or collection of) `Location` object(s) could be used here to replace all of these vars. Is there a way to tell Jackson "anytime you see `region` give me a `Location.region(region)`?"